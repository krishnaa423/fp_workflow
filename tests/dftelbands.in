&CONTROL
outdir='./tmp'
prefix='struct'
pseudo_dir='./ONCVPSP/sg15'
calculation='bands'
tprnfor=.true. 
/

&SYSTEM
ibrav=0
ntyp=1
nat=2
nbnd=14
ecutwfc=20.0
!noncolin=.true.
!lspinorb=.true. 
/

&ELECTRONS
/

&CELL
/

&IONS
/

CELL_PARAMETERS angstrom
   2.7150000000    2.7150000000    0.0000000000
   0.0000000000    2.7150000000    2.7150000000
   2.7150000000    0.0000000000    2.7150000000


ATOMIC_SPECIES
Si 28.085 Si_ONCV_PBE_sr.upf


ATOMIC_POSITIONS angstrom 
Si    0.0000000000    0.0000000000    0.0000000000
Si    1.3575000000    1.3575000000    1.3575000000


K_POINTS crystal
100
   0.5000000000    0.5000000000    0.5000000000 1.0
   0.4705882353    0.4705882353    0.4705882353 1.0
   0.4411764706    0.4411764706    0.4411764706 1.0
   0.4117647059    0.4117647059    0.4117647059 1.0
   0.3823529412    0.3823529412    0.3823529412 1.0
   0.3529411765    0.3529411765    0.3529411765 1.0
   0.3235294118    0.3235294118    0.3235294118 1.0
   0.2941176471    0.2941176471    0.2941176471 1.0
   0.2647058824    0.2647058824    0.2647058824 1.0
   0.2352941176    0.2352941176    0.2352941176 1.0
   0.2058823529    0.2058823529    0.2058823529 1.0
   0.1764705882    0.1764705882    0.1764705882 1.0
   0.1470588235    0.1470588235    0.1470588235 1.0
   0.1176470588    0.1176470588    0.1176470588 1.0
   0.0882352941    0.0882352941    0.0882352941 1.0
   0.0588235294    0.0588235294    0.0588235294 1.0
   0.0294117647    0.0294117647    0.0294117647 1.0
   0.0000000000    0.0000000000    0.0000000000 1.0
   0.0250000000    0.0000000000    0.0250000000 1.0
   0.0500000000    0.0000000000    0.0500000000 1.0
   0.0750000000    0.0000000000    0.0750000000 1.0
   0.1000000000    0.0000000000    0.1000000000 1.0
   0.1250000000    0.0000000000    0.1250000000 1.0
   0.1500000000    0.0000000000    0.1500000000 1.0
   0.1750000000    0.0000000000    0.1750000000 1.0
   0.2000000000    0.0000000000    0.2000000000 1.0
   0.2250000000    0.0000000000    0.2250000000 1.0
   0.2500000000    0.0000000000    0.2500000000 1.0
   0.2750000000    0.0000000000    0.2750000000 1.0
   0.3000000000    0.0000000000    0.3000000000 1.0
   0.3250000000    0.0000000000    0.3250000000 1.0
   0.3500000000    0.0000000000    0.3500000000 1.0
   0.3750000000    0.0000000000    0.3750000000 1.0
   0.4000000000    0.0000000000    0.4000000000 1.0
   0.4250000000    0.0000000000    0.4250000000 1.0
   0.4500000000    0.0000000000    0.4500000000 1.0
   0.4750000000    0.0000000000    0.4750000000 1.0
   0.5000000000    0.0000000000    0.5000000000 1.0
   0.5000000000    0.0250000000    0.5250000000 1.0
   0.5000000000    0.0500000000    0.5500000000 1.0
   0.5000000000    0.0750000000    0.5750000000 1.0
   0.5000000000    0.1000000000    0.6000000000 1.0
   0.5000000000    0.1250000000    0.6250000000 1.0
   0.5000000000    0.1500000000    0.6500000000 1.0
   0.5000000000    0.1750000000    0.6750000000 1.0
   0.5000000000    0.2000000000    0.7000000000 1.0
   0.5000000000    0.2250000000    0.7250000000 1.0
   0.5000000000    0.2500000000    0.7500000000 1.0
   0.5000000000    0.2666666667    0.7333333333 1.0
   0.5000000000    0.2833333333    0.7166666667 1.0
   0.5000000000    0.3000000000    0.7000000000 1.0
   0.5000000000    0.3166666667    0.6833333333 1.0
   0.5000000000    0.3333333333    0.6666666667 1.0
   0.5000000000    0.3500000000    0.6500000000 1.0
   0.5000000000    0.3666666667    0.6333333333 1.0
   0.5000000000    0.3833333333    0.6166666667 1.0
   0.5000000000    0.4000000000    0.6000000000 1.0
   0.5000000000    0.4166666667    0.5833333333 1.0
   0.5000000000    0.4333333333    0.5666666667 1.0
   0.5000000000    0.4500000000    0.5500000000 1.0
   0.5000000000    0.4666666667    0.5333333333 1.0
   0.5000000000    0.4833333333    0.5166666667 1.0
   0.5000000000    0.5000000000    0.5000000000 1.0
   0.4903846154    0.4903846154    0.5192307692 1.0
   0.4807692308    0.4807692308    0.5384615385 1.0
   0.4711538462    0.4711538462    0.5576923077 1.0
   0.4615384615    0.4615384615    0.5769230769 1.0
   0.4519230769    0.4519230769    0.5961538462 1.0
   0.4423076923    0.4423076923    0.6153846154 1.0
   0.4326923077    0.4326923077    0.6346153846 1.0
   0.4230769231    0.4230769231    0.6538461538 1.0
   0.4134615385    0.4134615385    0.6730769231 1.0
   0.4038461538    0.4038461538    0.6923076923 1.0
   0.3942307692    0.3942307692    0.7115384615 1.0
   0.3846153846    0.3846153846    0.7307692308 1.0
   0.3750000000    0.3750000000    0.7500000000 1.0
   0.3593750000    0.3593750000    0.7187500000 1.0
   0.3437500000    0.3437500000    0.6875000000 1.0
   0.3281250000    0.3281250000    0.6562500000 1.0
   0.3125000000    0.3125000000    0.6250000000 1.0
   0.2968750000    0.2968750000    0.5937500000 1.0
   0.2812500000    0.2812500000    0.5625000000 1.0
   0.2656250000    0.2656250000    0.5312500000 1.0
   0.2500000000    0.2500000000    0.5000000000 1.0
   0.2343750000    0.2343750000    0.4687500000 1.0
   0.2187500000    0.2187500000    0.4375000000 1.0
   0.2031250000    0.2031250000    0.4062500000 1.0
   0.1875000000    0.1875000000    0.3750000000 1.0
   0.1718750000    0.1718750000    0.3437500000 1.0
   0.1562500000    0.1562500000    0.3125000000 1.0
   0.1406250000    0.1406250000    0.2812500000 1.0
   0.1250000000    0.1250000000    0.2500000000 1.0
   0.1093750000    0.1093750000    0.2187500000 1.0
   0.0937500000    0.0937500000    0.1875000000 1.0
   0.0781250000    0.0781250000    0.1562500000 1.0
   0.0625000000    0.0625000000    0.1250000000 1.0
   0.0468750000    0.0468750000    0.0937500000 1.0
   0.0312500000    0.0312500000    0.0625000000 1.0
   0.0156250000    0.0156250000    0.0312500000 1.0
   0.0000000000    0.0000000000    0.0000000000 1.0

