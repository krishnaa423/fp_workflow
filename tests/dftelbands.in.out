
     Program PWSCF v.7.2 starts on 17Jul2024 at  5:10:24 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     28147 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input
Warning: card &CELL ignored
Warning: card / ignored
Warning: card &IONS ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     Atomic positions and unit cell read from directory:
     ./tmp/struct.save/
     Atomic positions from file used, from input discarded
 
     Message from routine setup:
     using ibrav=0 with symmetry is DISCOURAGED, use correct ibrav instead
 
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         295     295     91                 3287     3287     609
 
     Using Slab Decomposition
 


     bravais-lattice index     =            0
     lattice parameter (alat)  =       7.2558  a.u.
     unit-cell volume          =     270.1072 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=           14
     kinetic-energy cutoff     =      20.0000  Ry
     charge density cutoff     =      80.0000  Ry
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     GPU acceleration is ACTIVE.


     celldm(1)=   7.255773  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.707107   0.707107   0.000000 )  
               a(2) = (   0.000000   0.707107   0.707107 )  
               a(3) = (   0.707107   0.000000   0.707107 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  0.707107  0.707107 -0.707107 )  
               b(2) = ( -0.707107  0.707107  0.707107 )  
               b(3) = (  0.707107 -0.707107  0.707107 )  


     PseudoPot. # 1 for Si read from file:
     ./ONCVPSP/sg15/Si_ONCV_PBE_sr.upf
     MD5 check sum: 8553c464762ac42b85b1eed164b4d029
     Pseudo is Norm-conserving, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1510 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08500     Si( 1.00)

     48 Sym. Ops., with inversion, found (24 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Si  tau(   2) = (   0.3535534   0.3535534   0.3535534  )

     number of k points=   100

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:     3287 G-vectors     FFT dimensions: (  24,  24,  24)

     Estimated max dynamical RAM per process >       2.14 MB

     The potential is recalculated from file :
     ./tmp/struct.save/charge-density

     Starting wfcs are    8 randomized atomic wfcs +    6 random wfcs

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  1.25E-08,  avg # of iterations = 22.2

     total cpu time spent up to now is        9.1 secs

     End of band structure calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     Writing all to output data dir ./tmp/struct.save/
 
     init_run     :      0.01s CPU      0.06s WALL (       1 calls)
     electrons    :      7.84s CPU      8.91s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
                                        0.00s GPU  (       1 calls)
     potinit      :      0.00s CPU      0.04s WALL (       1 calls)
     hinit0       :      0.01s CPU      0.01s WALL (       1 calls)

     Called by electrons:
     c_bands      :      7.84s CPU      8.91s WALL (       1 calls)
     v_of_rho     :      0.00s CPU      0.03s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.03s CPU      0.03s WALL (     100 calls)
     init_us_2:gp :      0.02s CPU      0.03s WALL (     100 calls)
     cegterg      :      7.43s CPU      8.33s WALL (     146 calls)

     Called by *egterg:
     cdiaghg      :      2.75s CPU      3.13s WALL (    2316 calls)
                                        3.09s GPU  (    2316 calls)
     h_psi        :      2.73s CPU      3.14s WALL (    2462 calls)
                                        3.11s GPU  (    2462 calls)
     g_psi        :      0.01s CPU      0.05s WALL (    2216 calls)
                                        0.02s GPU  (    2216 calls)

     Called by h_psi:
     h_psi:calbec :      0.13s CPU      0.74s WALL (    2462 calls)
                                        0.70s GPU  (    2462 calls)
     vloc_psi     :      1.44s CPU      1.67s WALL (    2462 calls)
                                        1.63s GPU  (    2462 calls)
     add_vuspsi   :      0.37s CPU      0.48s WALL (    2462 calls)
                                        0.45s GPU  (    2462 calls)

     General routines
     calbec       :      0.08s CPU      0.11s WALL (    2462 calls)
     fft          :      0.00s CPU      0.00s WALL (      10 calls)
                                        0.00s GPU  (      10 calls)
     fftw         :      0.10s CPU      0.56s WALL (    4924 calls)
                                        0.49s GPU  (    4924 calls)
     davcio       :      0.01s CPU      0.11s WALL (     200 calls)
 
     Parallel routines
 
     PWSCF        :      7.95s CPU      9.62s WALL

 
   This run was terminated on:   5:10:34  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Warning: ieee_invalid is signaling
Warning: ieee_inexact is signaling
FORTRAN STOP
